cheroot/connections.py

```diff
@@ -0,0 +1,254 @@
+"""Utilities to manage open connections."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import io
+import select
+import six
```

Comment:
this is a third-party lib, it should go into a separate section

Reply:
Done.